### PR TITLE
ref(app-platform): Minor cleanup

### PR DESCRIPTION
**Pull request by MeredithAnya**
* Add `sentry.tasks.app_platform` to imports
* Use `window.location.assign` to send users to the `redirectUrl`
* Add default for `isAlertable` so it's not null

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx [hunk 1]
```diff
@@ -41,17 +41,18 @@ class SentryApplicationRow extends React.PureComponent {
       );
       const query = Object.assign(qs.parse(url.query), installQuery);
       redirectUrl = `${url.protocol}//${url.host}${url.pathname}?${qs.stringify(query)}`;
+      window.location.assign(redirectUrl);
     }
 
-    return redirectUrl;
+    browserHistory.push(redirectUrl);
```
Comment (mnoble) [inline src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx]: I would call this function something else now, if it's actually doing the redirecting.

**Review by mnoble — COMMENTED** [review]
Does using `browserHistory.push` with an external URL cause issues or something?

File: src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx (hunk 1 above)
Comment (MeredithAnya) [reply src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx]: yeah `browserHistory` only takes relative routes afaik and cool can rename

**Review by mnoble — APPROVED** [review]
👍

**Pull request merged**